### ISSUE TITLE
Feature request: accept an invitation

### ISSUE THREAD

**Issue by Marcool04** [8go/matrix-commander#6]
Hi,
First of all thanks a lot for this very helpful little utility!
I have created a specific matrix account on my homeserver for a matrix-commander instance, using another client, and have gone through the initial setup, with everything working ok. I can now send messages and read them.
I then invited the matrix-commander account into a room (with several other accounts), and wanted to send messages there, but these don't arrive, and it appears to be because I can't "accept" this invitation from the command line using matrix-commander, is this correct?
I guess if it is the case this issue could be seen as a feature-request. I feel matrix-commander is already so close to being a complete client, if this is not implemented yet it would be wonderful to have it.
Thanks again for all the great work.
Regards,
Mark.

**Comment by 8go** [comment]
Thanks @Marcool04 for raising the issue. 

Yes, the command-line cannot (currently) accept room invitations.

OK, you have created the commander account with another client (e.g. Element). I, BTW, have done exactly the same. So, you accept all room invitations from the Element account (and not from the command line). Same for issuing invitations to others. This (at least for me) is a very infrequent operation so there is no big issue doing it in the Element GUI.

Another option which is worth investigating, are the new features and options in the Synapse homeserver,yaml config file related to `auto-join`. Open your homeserver.yaml file and search for `auto-join`. You will find quite a few settings. Not sure if this is what you want or this suits you, but it might automate joining rooms for some people. I personally do not use `auto-join` so I have no concrete example I can give you.

In short, there is a solution (doing it in Element or another client), but I will keep the issue open to remind of this possible feature request.

Again, thanks @Marcool04  for your comment. Cheers.

**Comment by Marcool04** [comment]
Right, yes it's manageable to set up conversations from element desktop/web client and also accept invitations there. I don't use the `auto-join` options either, but indeed, they could be another solution.
Thanks for your fast response anyhow! Regards.

**Comment by 8go** [comment]
There was a lovely contribution from @berlincount  in PR #8 
Have a look at that. This will or might help you @Marcool04 
Now the matrix-commander CLI can a) create a room and then b) send an invitation.

**Comment by 8go** [comment]
Hi @Marcool04 , I added the room_join (=accept invitation) feature today, And room_invite was added yesterday.

See d9ea160f950621a4c23fc19a8638ccfc83d40194 

So, this issue and this feature request is closed. 

Download the new version to make use of the new functionality!

**Comment by Marcool04** [comment]
Wow @8go thanks so much for all the hard work! You've expanded the programs capabilities a load with those last few commits! Fantastic 👍
Cheers to all who helped out and to you.
Keep up the good work!
